metagraph/tests/test_bit_vector.cpp

```diff
@@ -573,6 +574,113 @@ TEST(bit_vector_sd, CheckIfInverts) {
     }
 }
 
+sdsl::sd_vector_disk<>
+construct_sd_vector_disk(sdsl::bit_vector bv, const std::string &fname, size_t offset) {
+    sdsl::sd_vector_disk_builder builder(bv.size(), sdsl::util::cnt_one_bits(bv), fname, offset);
+    call_ones(bv, [&](uint64_t i) { builder.set(i); });
+    return sdsl::sd_vector_disk<>(builder);
+}
+
+void compare(const sdsl::sd_vector<> &first, const sdsl::sd_vector_disk<> &second) {
+    ASSERT_EQ(first.size(), second.size());
+
+    sdsl::sd_vector<>::select_1_type first_slct(&first);
+    sdsl::sd_vector<>::rank_1_type first_rank(&first);
+
+    sdsl::sd_vector_disk<>::select_1_type second_slct(&second);
+    sdsl::sd_vector_disk<>::rank_1_type second_rank(&second);
+
+    uint64_t m = first_rank(first.size());
```

Comment:
maybe also add a test here to make sure that `first_rank(first.size()) == second_rank(second.size())`?